gh-codeql

```diff
@@ -92,10 +92,23 @@ if [ "$1" = "set-channel" ]; then
     exit 0
 fi
 
+
+function update_latest {
+    if [ "$channel" = "release" ]; then
+        # For the release channel, we sort by semantic version order since the tags correspond to release versions.
+        # We ignore draft releases and pre-release versions in this case since we want to give the user a stable version.
+        latest=$(gh api "repos/$repo/releases" --paginate --jq ".[] | select(.draft == false and .prerelease == false) | .tag_name" | sort -V | tail -1)
+    else
+        # For the nightly channel, we just take the latest release since the tags there are not semantic versions.
+        latest=$(gh api "repos/$repo/releases" --paginate --jq ".[].tag_name" | head -1)
```

Comment:
Are we guaranteed that tags are returned in chronological order?

If so, you probably don't need `--paginate` here. If not, we need a better mechanism for picking the latest version.

Reply:
Indeed, the releases are ordered chronologically and we can remove `--paginate`.